services/publish.webmaker.org/lib/remix.js

```diff
@@ -22,10 +24,27 @@ function injectMetadata(html, metadata) {
 }
 
 function injectRemixScript(html) {
-  return html.replace(
-    /<\/head/,
-    `<script src="${REMIX_SCRIPT}" type="text/javascript"></script>\n$&`
-  );
+  // Check if <head> tag exists
+  var retInject = ``;
+  const dom = new JSDOM(html);
+
+  // If they exist, inject remix script before closing </head> tag
+  if (dom.window.document.querySelector(`head`)) {
```

Comment:
See https://github.com/mozilla/thimble.mozilla.org/issues/2312.  We should be able to require a newer node.js for Thimble, and isolate what we use for publish in Vagrant.